account_invoice_mgmt_atm/views/res_users_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_res_users_form_ATM_config" model="ir.ui.view">
+        <field name="name">res.users (ATM)</field>
+        <field name="model">res.users</field>
+        <field name="inherit_id" ref="base.view_users_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='messaging']" position="after">
+                <group name="atm_config" string="ATM Configuration">
```

Comment:
This data should be only accessed by a user with permissions in Accounting features (eg. see https://github.com/odoo/odoo/blob/4cc086d330b73514fbc64a7fcf22d8a7a9f1b691/addons/account/security/ir.model.access.csv#L37)